genmc-sys/src/lib.rs

```diff
@@ -120,6 +132,19 @@ mod ffi {
         Debug3ReadsFrom,
     }
 
+    #[derive(Debug)]
+    /// Setting to control which execution graphs GenMC prints after every execution.
+    enum ExecutiongraphPrinting {
+        /// Print no graphs.
+        None,
```

Comment:
Can you use attributes here or does cxx not like then? If you can, you can avoid the manual `Default` impl by using https://doc.rust-lang.org/std/default/trait.Default.html#enums

Reply:
Bummer.